### DISCUSSION TITLE
Span events are not visible in logs (used LoggingExporter)

### DISCUSSION THREAD

**Discussion by mamathadaram** [open-telemetry/opentelemetry-java#4874] [Q&A]
I am using LoggingExporter and unable to see the span events in the logs only the number of events recorded are being logged.
But able to see the events in Zipkin UI.

Are span events not supported with Logging exporter?

**Comment by jkwatson** [comment]
The logging exporter is only for very simplistic debug logging. It's not intended to have all the span data in it, and is documented as such.

**Reply by mamathadaram** [reply]
ok. thank you
What does the relative time indicate for the span event

<img width="281" alt="image" src="https://user-images.githubusercontent.com/17672527/197187455-6c5c03aa-75aa-4c8b-abdb-39df17368935.png">

**Reply by jkwatson** [reply]
Is this something in the zipkin UI? I would guess (just a guess) that this is the number of milliseconds after the start of span that the event took place.

**Reply by mamathadaram** [reply]
yes , its from Zipkin UI
